templates/blog/blog_post.html

```diff
@@ -8,9 +8,9 @@
     <div class="content-width grid-standard page-layout page-layout--without-sidenav">
         <div class="page-layout__main-content">
             <div class="streamfields-wrapper">
-                {% if page.body %}
-                    {% include_block page.body %}
-                {% endif %}
+                {% for block in page.body %}
+                    {% include_block block %}
+                {% endfor %}
```

Comment:
This is the method that stops rendering the auto-generated wrapping divs. I have applied it everywhere we loop over streamfields.